### PR TITLE
Add autofix to at-rule-semicolon-newline-after

### PR DESCRIPTION
**Pull request by modosc**
> Which issue, if any, is this issue related to?

#2829

> Is there anything in the PR that needs further explanation?

this was originally #3146 but i accidentally deleted the remote of the branch so i'm reopening it here.

### CODE REVIEW

File: lib/rules/at-rule-semicolon-newline-after/__tests__/index.js [hunk 1]
```diff
@@ -79,42 +79,49 @@ testRule(rule, {
...
       message: messages.expectedAfter(),
-      line: 1,
-      column: 18
+      line: 2,
+      column: 12
     },
     {
-      code: "a{\n@extend .b; @extend .c\n}",
+      code: "a{\r\n@extend .b; @extend .c\n}",
+      fixed: "a{\r\n@extend .b;\r\n @extend .c\n}",
```
Comment (hudochenkov) [inline lib/rules/at-rule-semicolon-newline-after/__tests__/index.js]: Should be `\r\n` at the very end in `code`, and `fixed`.

**Comment by jeddy3** [comment]
Yep, looks good. Once @hudochenkov minor requests are in I think we can merge this.

It's amazing to see so many autofix PRs in the works! It'd be great if we can roll these all out in the next release.

**Comment by modosc** [comment]
> It's amazing to see so many autofix PRs in the works! It'd be great if we can roll these all out in the next release.

i'm happy someone else picked this up - i've been out of the js world for a while at my newest job so i haven't had time or excuse to keep working on them.

**Comment by jeddy3** [comment]
> i'm happy someone else picked this up - i've been out of the js world for a while at my newest job

Ditto. I've mainly been doing UX stuff the last 6 months and JavaScript feels a bit foreign :)

**Comment by jeddy3** [comment]
> -   Added: `at-rule-semicolon-newline-after` autofix ([#3450](https://github.com/stylelint/stylelint/pull/3450)).

**Pull request merged**